pageserver/src/tenant/timeline.rs

```diff
@@ -1614,7 +1614,7 @@ impl Timeline {
         let (conf, tenant_id, timeline_id) = (self.conf, self.tenant_id, self.timeline_id);
         let span = tracing::Span::current();
 
-        let (loaded_layers, needs_upload, total_physical_size) = tokio::task::spawn_blocking({
+        let (loaded_layers, sync, total_physical_size) = tokio::task::spawn_blocking({
```

Comment:
`sync` is a bit ambiguous of a name, maybe `to_sync` is better (if `needs_sync` is too long).

Reply:
with 4 elements it was too long for a line `to_sync` fits, b45df96c2